guides/plugins/plugins/content/stock/implementing-your-own-stock-storage.md

```diff
@@ -0,0 +1,124 @@
+# Implementing your own stock storage
+
+## Overview
+
+Shopware stores stock as simple integer values in the `product` table. If you need a more advanced stock management system or would like to write the stock alterations to a different system, you can implement your own stock storage.
+
+## Prerequisites
+
+Here you will be decorating a service; therefore, it will be helpful to familiarise yourself with the [Adjusting a Service](../../plugin-fundamentals/adjusting-service) guide.
```

Comment:
```suggestion
Here you will be decorating a service; therefore, it will be helpful to familiarise yourself with the [Adjusting a Service](../../../../../guides/plugins/plugins/plugin-fundamentals/adjusting-service.md) guide.
```